docs/tutorials/quorum-dev-quickstart/using-the-quickstart.md

```diff
@@ -414,6 +415,123 @@ All nodes are validating the same blockchain of transactions, with the private t
+that you can run
+
+```bash
+node ./scripts/deploy.js
+```
+
+When this is completed, an output `permission-config.json` is created which contains the addresses of the
+contract and any account addresses that will serve as admins. In this example we've used all accounts, if you
+deploy these contracts in a production network, please select only those which need to be admins.
+
+The file looks like this (but will have different addresses)
+
+```bash
+{
+    "permissionModel": "v2",
+    "upgradableAddress": "0x1932c48b2bf8102ba33b4a6b545c32236e342f34",
+    "interfaceAddress": "0x4d3bfd7821e237ffe84209d8e638f9f309865b87",
+    "implAddress": "0xfe0602d820f42800e3ef3f89e1c39cd15f78d283",
+    "nodeMgrAddress": "0x8a5e2a6343108babed07899510fb42297938d41f",
+    "accountMgrAddress": "0x9d13c6d3afe1721beef56b55d303b09e021e27ab",
+    "roleMgrAddress": "0x1349f3e1b8d71effb47b840594ff27da7e603d17",
+    "voterMgrAddress": "0xd9d64b7dc034fafdba5dc2902875a67b5d586420",
+    "orgMgrAddress" : "0x938781b9796aea6376e40ca158f67fa89d5d8a18",
+    "nwAdminOrg": "ADMINORG",
+    "nwAdminRole" : "ADMIN",
+    "orgAdminRole" : "ORGADMIN",
+    "accounts":["0xed9d02e382b34818e88b88a309c7fe71e65f419d", "0xca843569e3427144cead5e4d5999a3d0ccf92b8e", ....],
+    "subOrgBreadth" : 3,
+    "subOrgDepth" : 4
+}
+```
+
+where:
+
+* permissionModel - Permission model to be used (v1 or v2).
+* upgradableAddress- Address of deployed contract PermissionsUpgradable.sol.
+* interfaceAddress - Address of deployed contract PermissionsInterface.sol.
+* implAddress - Address of deployed contract PermissionsImplementation.sol.
+* nodeMgrAddress - Address of deployed contract NodeManager.sol.
+* accountMgrAddress - Address of deployed contract AccountManager.sol.
+* roleMgrAddress - Address of deployed contract RoleManager.sol.
+* voterMgrAddress - Address of deployed contract VoterManager.sol.
+* orgMgrAddress - Address of deployed contract OrgManager.sol.
+* nwAdminOrg - Name of the initial organization to be created as a part of the network boot up
+with a new permissions model. This organization owns all the initial nodes and network
+administrator accounts at the network boot up.
+* nwAdminRole - Role ID to be assigned to the network administrator accounts.
+* orgAdminRole - Role ID to be assigned to the organization administrator account.
+* accounts - Initial list of accounts linked to the network administrator organization and assigned
+the network administrator role. These accounts have complete control of the network and can propose
+and approve new organizations into the network.
+* subOrgBreadth - Number of sub-organizations that any organization can have.
+* subOrgDepth - Maximum depth of sub-organization hierarchy allowed in the network.
+
+The last step is to copy the above `permission-config.json` into `data` directory of each GoQuorum node and
+restart the nodes. This can be done by running the script
+
+```bash
+./scripts/restartNetwork.sh
+```
```

Comment:
Can this be put in the root like `./run.sh` `./resume.sh` etc

Reply:
Nice, this is a really good idea. Have updated the quickstart to have this now